website/docs/guides/docs/versioning.md

```diff
@@ -259,7 +259,7 @@ The [@hello](hello.md#paginate) document is great!
 See the [Tutorial](../getting-started/tutorial.md) for more info.
 ```
 
-### Global or versioned colocated assets {#global-or-versioned-colocated-assets}
+### Global or versioned collocated assets {#global-or-versioned-collocated-assets}
```

Comment:
"collocate" is different from "co-locate", I guess? Maybe add a hyphen here instead?

Reply:
https://www.merriam-webster.com/dictionary/collocate seems to be correct

Reply:
Okay that makes sense